docs/api/ref-cheatsheet.md

```diff
@@ -39,15 +39,45 @@ add_labels
 add_brands
 ```
 
+### Adding nutrition facts for the prepared product
+You can send prepared nutritional values
+* nutriment_energy-kj (regular)
+* nutriment_energy-kj_prepared (prepared)
+
 ## Search for Products
 
 **Important:** full text search currently works only for v1 API (or search-a-licious, which is in beta)
 
-[documentation for v1 Search API](https://wiki.openfoodfacts.org/API/Read/Search)
-
-[Reference documentation for v2 search API](https://openfoodfacts.github.io/openfoodfacts-server/api/ref-v2/#get-/api/v2/search)
-
-The future of search is the [search-a-licious project](https://github.com/openfoodfacts/search-a-licious), deployed, in beta, at [search.openfoodfacts.org](https://search.openfoodfacts.org/).
+* [Documentation for v1 Search API](https://wiki.openfoodfacts.org/API/Read/Search)
+
+* [Reference documentation for v2 search API](https://openfoodfacts.github.io/openfoodfacts-server/api/ref-v2/#get-/api/v2/search)
+
+* The future of search in Open Food Facts is the [search-a-licious project](https://github.com/openfoodfacts/search-a-licious), deployed, in beta, at [search.openfoodfacts.org](https://search.openfoodfacts.org/). It has an API: [Search-a-licious API](https://search.openfoodfacts.org/docs)
+
+## Get suggestions for fields
+### New solution: Search-a-licious (has all actually used values)
```

Comment:
We have a v3 suggestions: https://openfoodfacts.github.io/openfoodfacts-server/api/ref-v3/#get-/api/v3/taxonomy_suggestions

It has features to get suggestions from synonyms, suggestions for a specific category and packaging shape etc.

e.g. https://world.openfoodfacts.org/api/v3/taxonomy_suggestions?tagtype=labels&lc=fr&string=f&get_synonyms=1

https://world.openfoodfacts.org/api/v3/taxonomy_suggestions?tagtype=packaging_materials&shape=box